src/ExtensionsSample/Program.cs

```diff
@@ -44,7 +45,22 @@ public static void Main(string[] args)
                 ToAddress = "admin@webjobssamples.com",
                 FromAddress = new MailAddress("samples@webjobssamples.com", "WebJobs Extensions Samples")
             };
-            config.UseSendGrid(sendGridConfiguration);
+            //config.UseSendGrid(sendGridConfiguration);
+
+            if (File.Exists("dropbox-connectionkey.txt"))
```

Comment:
Perhaps we should discuss your config needs so I can understand what you need to accomplish. It looks like this config block is a TODO, since it contains specific subs, etc.